pom.xml

```diff
@@ -276,6 +276,12 @@
                     <version>2.8.1</version>
                     <configuration>
                         <argLine>${argLine} -Djava.awt.headless=true -noverify</argLine>
+                        <includes>
+                            <include>**/*.java</include>
```

Comment:
Looks like this is a known issues: http://stackoverflow.com/a/34308282.

Could we do this instead? I tried and it seems to work:
```
<includes>
  <include>**/*Test.java</include>
  <include>**/Test*.java</include>
  <include>**/*TestCase.java</include>
  <include>**/*Spec.class</include>
```

Don't know if that's the plan but we should update all the other projects as well (e.g., Alliance).